Manito/Manito/Screens/Letter/LetterViewController.swift

```diff
@@ -176,6 +204,15 @@ final class LetterViewController: BaseViewController {
     
     // MARK: - func
     
+    private func setupEmptyView() {
+        if letterList.isEmpty {
+            emptyLabel.isHidden = false
+        }
+        else {
+            emptyLabel.isHidden = true
+        }
```

Comment:
p5;

이렇게 나타낼 수도 있겠군요.
```suggestion
        emptyLabel.isHidden = !letterList.isEmpty
```

Reply:
[5302c54](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/258/commits/5302c540fff53af822cbda3ee491171ed28df6b8)
대박사건..!!